code/Database.md

```diff
@@ -111,4 +111,4 @@ To modify the database the following steps must be completed:
 3. Add a new `testdata/*.v<N>.sql`, where `N` is the specific migration step.
 4. If there are new tables, then add a `-- NEW DATA --` and insert rows to the new tables.
 
-See [How to add a new table to Satellite.DB](Satellite-how-to-new-table.md) for more detailed instructions.
\ No newline at end of file
+See [How to add a new table to Satellite.DB](Satellite-how-to-new-table) for more detailed instructions.
```

Comment:
thanks for fixing

Reply:
It depends, what we choose, I'm waiting for a response.